db/migrations/2025_04_15_112546_add_publishing_related_states_to_studies.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime, UTC
+
+import sqlalchemy as sql
+
+
+def up(conn):
+    queries = [
+        "ALTER TABLE Study ADD COLUMN publishableAt    datetime",
+        "ALTER TABLE Study ADD COLUMN publishedAt      datetime",
+        "ALTER TABLE Study ADD COLUMN embargoExpiresAt datetime",
```

Comment:
The embargo setup is not implemented yet, but I decided to add the column anyway. I may drop it if we decide not to have an explicit control for it.